tests/metrics/network/network-metrics-iperf3.sh

```diff
@@ -42,10 +42,12 @@ function setup {
 function iperf3_bandwidth {
 	setup
 	bandwidth_result=$(mktemp)
-	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s" > /dev/null && \
-	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
-	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -t ${time}" > "$bandwidth_result"
-	total_bandwidth=`cat $bandwidth_result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}'`
+	server_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s"
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "${server_command}" > /dev/null && \
```

Comment:
@devimc to avoid that the client and the server goes to connection timeout as everything must be synchoronized